src/forecaster/forecaster.py

```diff
@@ -37,8 +37,8 @@ def forecast(course_enrolment: dict, program_enrolment: dict, schedule: dict, fo
         schedule_file = open(os.path.join(os.path.dirname(__file__), 'static_files/testSchedule.json'))
         schedule  = json.load(schedule_file)
         schedule_file.close()
-
-
+    else:
```

Comment:
Shouldn't this else be on the if statement for the program enrolment file, not the schedule file? Or do I misunderstand?

Reply:
It is a separate else condition, I think

Reply:
Oh I see what you mean, one second